tests/unit/dom.ts

```diff
@@ -237,5 +237,169 @@ registerSuite({
+			return {
+				addClass: {
+					afterEach() {
+						targetElement.className = '';
+					},
+
+					'add single class'() {
+						assert.notInclude(targetElement.className, 'test');
+						dom.addClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+					},
+
+					'add multiple classes'() {
+						dom.addClass(targetElement, 'test1', 'test2');
+						assert.include(targetElement.className, 'test1');
+						assert.include(targetElement.className, 'test2');
+					},
+
+					'null node'() {
+						assert.doesNotThrow(function () {
+							dom.addClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.addClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.addClass(targetElement);
+						});
+					},
+
+					'existing class'() {
+						dom.addClass(targetElement, 'test');
+						dom.addClass(targetElement, 'test');
+						assert.lengthOf(targetElement.className.match(/test/), 1);
+					}
+				},
+
+				containsClass: {
+					afterEach() {
+						targetElement.className = '';
+					},
+
+					'contains single class'() {
+						dom.addClass(targetElement, 'test');
+						assert.isTrue(dom.containsClass(element, 'test'));
+					},
+
+					'null node'() {
+						assert.doesNotThrow(function () {
+							dom.containsClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.containsClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.containsClass(targetElement, null);
+						});
+					}
+				},
+
+				removeClass: {
+					afterEach() {
+						targetElement.className = '';
+					},
+
+					'remove single class'() {
+						dom.addClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+						dom.removeClass(targetElement, 'test');
+						assert.notInclude(targetElement.className, 'test');
+					},
+
+					'remove multiple classes'() {
+						dom.addClass(targetElement, 'test1', 'test2');
+						assert.include(targetElement.className, 'test1');
+						assert.include(targetElement.className, 'test2');
+						dom.removeClass(targetElement, 'test1', 'test2');
+						assert.notInclude(targetElement.className, 'test1');
+						assert.notInclude(targetElement.className, 'test2');
+					},
+
+					'null node'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.removeClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(targetElement);
+						});
+					},
+
+					'remove nonexistent class'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(targetElement, 'random');
+						});
```

Comment:
It would probably also be a good idea to add some unrelated classes to the element and make sure they still exist after this (i.e. no adverse side effects).

I'd also probably just combine the multiple test below into this one (still in a separate `doesNotThrow`), and you might also add one that tries to remove one class that does exist and one class that doesn't at the same time.